R/utils.R

```diff
@@ -1,3 +1,19 @@
+recodeValues <- function(values, desiredValues, dataType) {
```

Comment:
This function name for the categorical overlay didn't make sense to me. I expected swapping values for desiredValues like in a derived var. Maybe something like "filterValues" or "simplifyOverlayValues" would be a little more clear?

Reply:
so some values get recoded to __Unselected__ , and for continuous vars i think they will become strings (bin labels) where they were numbers and dates. so its a bit hard to name well. but ill think about it